src/Storages/ObjectStorage/registerStorageObjectStorage.cpp

```diff
@@ -63,7 +63,7 @@ createStorageObjectStorage(const StorageFactory::Arguments & args, StorageObject
 
     return std::make_shared<StorageObjectStorage>(
         configuration,
-        configuration->createObjectStorage(context, /* is_readonly */ false),
+        configuration->createObjectStorage(context, /* is_readonly */ args.mode != LoadingStrictnessLevel::CREATE),
```

Comment:
Please add a comment. What does `is_readonly` actually mean? (looks like we use this flag only for azure, maybe we should rename the flag)